apps/sms/js/selection_handler.js

```diff
@@ -89,20 +109,18 @@ SelectionHandler.prototype = {
   // and if all messages are checked : deselect them all.
   toggleCheckedAll: function sel_toggleCheckedAll() {
     var selected = this.selected.size;
```

Comment:
I don't really like this var name, maybe you can rename it to "selectedSize", or even get rid of it?